servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -220,7 +220,7 @@ public void commitBlock(long userId, long blockId) throws AlreadyExistsException
     } catch (TException te) {
       throw new IOException("Failed to commit block to master.", te);
     } finally {
-      mBlockStore.unlockBlock(userId, blockId);
+      mBlockStore.unlockBlock(lockId);
```

Comment:
Currently two methods are equivalent in function. But `mBlockStore.unlockBlock(userId, blockId);` is added tentatively and deprecated and `mBlockStore.unlockBlock(lockId);` is the preferred way?